apps/site/src/app/(main)/portal/@applicant/components/ReturnHome.tsx

```diff
@@ -0,0 +1,14 @@
+import Button from "@/lib/components/Button/Button";
+
+export default function SignWaiver() {
```

Comment:
Issue: `SignWaiver` is already a component defined elsewhere, so naming this `ReturnHome` will reduce confusion